app/controllers/sitemaps_controller.rb

```diff
@@ -36,4 +50,8 @@ def set_surrogate_controls(date)
       @stale_while_revalidate = "432000" # five days
     end
   end
+
+  def offset
+    params[:sitemap].split("-")[2].to_i * RESULTS_LIMIT # elvaluates to 0 if not present or not a number
```

Comment:
There's a typo in the comment, it should read "evaluates". But maybe the code should just be a bit clearer so we don't need a comment at all.